wagtailvideos/models.py

```diff
@@ -233,6 +234,26 @@ class Video(AbstractVideo):
         'tags',
     )
 
+def get_video_model():
+    from django.conf import settings
+    from django.apps import apps
+
+    try:
+        app_label, model_name = settings.WAGTAILMEDIA_VIDEO_MODEL.split('.')
```

Comment:
looks like this is jacked straight from wagtailmedia https://github.com/torchbox/wagtailmedia/blob/9a4112d81783cc001cb2baa19bea899222c03ed0/wagtailmedia/models.py#L124

This package wasn't build with the ability to override the model used for videos.

There's a lot of places where references to the `Video` models is used directly, so giving users the ability to override without fixing those references doesn't make sense.

Anywhere that `get_video_model` is used you can replace with a reference to `Video` instead. Feel free to open a pull request if you need the ability to override the `Video` model, but its out of scope for upgrading compatibility to 2.4